lib/resend/mailer.rb

```diff
@@ -0,0 +1,39 @@
+require "resend"
+
+module Resend
+  class Mailer
+
+    attr_accessor :config, :settings
+
+    def initialize(config)
+      @config = config
+      raise Resend::ResendError.new("Config requires api_key", @config) unless @config.has_key?(:api_key)
+      @settings = { return_response: true } # avoids NilError exception
+      @resend_client = Resend::Client.new config[:api_key]
+    end
+
+    def deliver!(mail)
+      params = {
+        from: mail[:from].to_s,
+        to: mail.to,
+        subject: mail.subject,
+      }
+      params[:cc] = mail[:cc] if mail[:cc].present?
+      params[:bcc] = mail[:bcc] if mail[:bcc].present?
+      params[:reply_to] = mail[:reply_to] if mail[:reply_to].present?
+      params[:html] = mail.body.decoded
```

Comment:
I ran into a validation error with ActiveMailer. From the logs on my dashboard, the request was sending an empty `html` param.

Making this change fixed it and messages were sending properly.

```ruby
params[:html] = mail.html_part.decode_body
```